### PR TITLE
update to audacity 2.3.0

### PR DESCRIPTION
**Pull request by eszlari**

### CODE REVIEW

File: org.audacityteam.Audacity.yaml [hunk 1]
```diff
@@ -1,10 +1,11 @@
 app-id: org.audacityteam.Audacity
 branch: stable
 runtime: org.freedesktop.Platform
-runtime-version: 1.6
+runtime-version: '1.6'
```
Comment (jurf) [inline org.audacityteam.Audacity.yaml]: Hm, weird, never noticed the missing quotes. Anyway, might as well switch to 18.08.

**Comment by jurf** [comment]
bot, build

**Comment by jurf** [comment]
Just checking, have you actually tested this on your machine?

**Comment by eszlari** [comment]
Yes, I tested the build locally.

**Pull request merged**